react/features/base/toolbox/components/ToolboxItem.native.js

```diff
@@ -39,13 +39,18 @@ export default class ToolboxItem extends AbstractToolboxItem<Props> {
         const {
             disabled,
             elementAfter,
+            hideIcon,
```

Comment:
Can't you use the new `Button` component instead of modifying these?

Reply:
Fixed.